build-support/bin/publish_docs.sh

```diff
@@ -4,21 +4,38 @@
 
 set -eo pipefail
 
+PANTS_GH_PAGES='https://github.com/pantsbuild/pantsbuild.github.io.git'
+GIT_URL="${GIT_URL:-$PANTS_GH_PAGES}"
+
+PANTS_SITE_URL='https://pantsbuild.github.io'
```

Comment:
I think this part of the script predates https://www.pantsbuild.org/ . Probably makes sense to say https://www.pantsbuild.org/ nowadays (tho I wouldn't be shocked if someone who knows the magic dns/redir/something behind that tells us that's a terrible idea)

Reply:
I was going to make that change, also wasn't sure if there was some magic that would break if I changed it. Will let others weigh in, but I've changed it to https://www.pantsbuild.org/ in af8de96. Thanks!